src/content/en/tools/workbox/guides/_shared/base-schema.html

```diff
@@ -0,0 +1,229 @@
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>globFollow</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Boolean</code>, defaulting to <code>true</code></em>
+    </p>
+    <p>
+      Determines whether or not symlinks are followed when generating the precache manifest.
+    </p>
+    <p>
+      For more information, see the definition of <code>follow</code> in the <code>glob</code>
+      <a href="https://github.com/isaacs/node-glob#options" class="external">documentation</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globFollow: false</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>globIgnores</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Array</code> of <code>String</code>, defaulting to
+        <code>['node_modules/**/*']</code></em>
+    </p>
+    <p>
+      A set of patterns matching files to always exclude when generating the precache manifest.
+    </p>
+    <p>
+      For more information, see the definition of <code>ignore</code> in the <code>glob</code>
+      <a href="https://github.com/isaacs/node-glob#options" class="external">documentation</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globIgnores: ['**/ignored.html']</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>globPatterns</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Array</code> of <code>String</code>, defaulting to
+        <code>['**/*.{js,css,html}']</code></em>
+    </p>
+    <p>
+      Files matching against any of these patterns will be included in the precache manifest.
+    </p>
+    <p>
+      For more information, see the
+      <a href="https://github.com/isaacs/node-glob#glob-primer" class="external">glob primer</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globPatterns: ['dist/*.{js,png,html,css}']</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>globStrict</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Boolean</code>, defaulting to <code>true</code></em>
+    </p>
+    <p>
+      If <code>true</code>, an error reading a directory when generating a precache manifest will
+      cause the build to fail. If <code>false</code>, the problematic directory will be skipped.
+    </p>
+    <p>
+      For more information, see the definition of <code>strict</code> in the <code>glob</code>
+      <a href="https://github.com/isaacs/node-glob#options" class="external">documentation</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globStrict: false</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>manifestTransforms</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Array</code> of <code>ManifestTransform</code>, defaulting to
+        <code>null</code></em>
+    </p>
+    <p>
+      One or more
+      <a href="/web/tools/workbox/reference-docs/prerelease/module-workbox-build.html#.ManifestTransform"><code>ManifestTransform</code></a>
```

Comment:
I think you might need to drop the .html for this to work

Reply:
technically you can keep it, but it results in an additional redirect and doesn't play nicely with analytics. so, yah, please remove ;)

Reply:
Done.